user-guide/Advanced_Modules/Dashboards_app/Available_visualizations/Tables/Table_component.md

```diff
@@ -104,7 +106,9 @@ To apply a **filter based on a specific column**:
 To **sort the table**, you can click a column header.
 
 - To toggle between ascending and descending order, click the column header again.
-- To apply additional sorting, press Ctrl while clicking one or more additional headers. The first column will then be used for the initial sorting, the next one to sort equal values of the first column, and so on.
+
+- To apply additional sorting, press Ctrl while clicking one or more additional headers. The first column will then be used for the initial sorting, the next one to sort equal values of the first column, and so on. Note that this is only fully supported from DataMiner 10.2.12/10.3.0 onwards.
```

Comment:
Sorting on multiple columns in a table has been supported since RN28010, this was all done client side. RN34526 enhanced this feature to forward the sorts to the server which should improve performance since sorting won't happen in the client anymore. _(Client sorting requires all data to be send to the client while server side sorting only sends the sorted pages to the client)_

Reply:
Thanks for the feedback! I'll adjust this and also clarify it in the release notes.